third_party/toolchains/bazel_0.23.2/BUILD

```diff
@@ -0,0 +1,188 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This becomes the BUILD file for @local_config_cc// under Windows.
```

Comment:
to get some more context, where's the contents of bazel_0.23.2 directory coming from?  depending on the answer, it might be a good idea to add a README.md file under third_party/toolchains/bazel_0.23.2 that explains what is the the context: purpose of those files, why they were copied from external repo etc. Are the files defining the windows toolchain to stay or it is just temporary?

Reply:
renamed folder for clearance